package.json

```diff
@@ -26,26 +20,35 @@
     "eslint": "^4.0.0",
     "eslint-plugin-json": "^1.2.0",
     "eslint-plugin-mozilla": "^0.4.0",
+    "fixpack": "^2.3.1",
     "shield-study-schemas": "^0.8.3",
     "webpack": "^2.6.1"
   },
-  "bugs": {
-    "url": "https://github.com/mozilla/shield-studies-addon-utils/issues"
-  },
   "files": [
     "dist"
   ],
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/mozilla/shield-studies-addon-utils.git"
-  },
+  "homepage": "https://github.com/mozilla/shield-studies-addon-utils#readme",
   "keywords": [
-    "mozilla",
     "addon",
-    "shield",
-    "shield-study",
+    "jsm",
+    "mozilla",
     "normandy",
-    "jsm"
+    "shield",
+    "shield-study"
   ],
-  "homepage": "https://github.com/mozilla/shield-studies-addon-utils#readme"
+  "license": "MPL-2.0",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mozilla/shield-studies-addon-utils.git"
+  },
+  "scripts": {
+    "build-test-addon-xpi": "./bin/make_xpi.sh",
+    "dist": "webpack",
+    "eslint": "eslint src --ext jsm --ext js --ext json",
+    "predist": "npm run eslint",
+    "prepack": "fixpack && npm run dist",
+    "pretest": "npm run dist && cp -rp dist/StudyUtils.jsm test-addon/ && npm run build-test-addon-xpi",
```

Comment:
I am not sure if we need both the `cp -rp dist/StudyUtils.jsm test-addon/` in `pretest` and also `cp dist/StudyUtils.jsm test-addon/` in the `make_xpi.sh` script. What do you think?